metricbeat/module/zookeeper/server/server_integration_test.go

```diff
@@ -48,7 +48,7 @@ func TestFetch(t *testing.T) {
 		metricsetFields := event.MetricSetFields
 
 		// Check values
-		assert.Equal(t, "06/29/2018 04:05 GMT", metricsetFields["version_date"])
+		assert.Equal(t, "02/06/2016 03:18 GMT", metricsetFields["version_date"])
```

Comment:
I think it's because I was using the official image with `3.4.13` instead of the unofficial configured one in the Dockerfile `jplock/zookeeper:3.4.8`.

But, yes, we can just compare other fields and maintain current image.

Reply:
But why was the build green here? https://github.com/elastic/beats/pull/10341

Reply:
I'm wondering the same, also with this error https://github.com/elastic/beats/pull/10375#discussion_r252190299